tests/Unit/Numerical/Spectral/Test_DefiniteIntegral.cpp

```diff
@@ -97,9 +98,12 @@ TEST_CASE("Unit.Numerical.Spectral.DefiniteIntegral", "[Functors][Unit]") {
       test_definite_integral_2d(Index<2>(n0, n1));
     }
   }
-  for (size_t n0 = min_extents; n0 <= std::min(6_st, Basis::lgl::maximum_number_of_pts); ++n0) {
-    for (size_t n1 = min_extents; n1 <= std::min(7_st, Basis::lgl::maximum_number_of_pts); ++n1) {
-      for (size_t n2 = min_extents; n2 <= std::min(8_st, Basis::lgl::maximum_number_of_pts); ++n2) {
+  for (size_t n0 = min_extents;
+       n0 <= std::min(6_st, Basis::lgl::maximum_number_of_pts); ++n0) {
+    for (size_t n1 = min_extents;
+         n1 <= std::min(7_st, Basis::lgl::maximum_number_of_pts); ++n1) {
+      for (size_t n2 = min_extents;
+           n2 <= std::min(8_st, Basis::lgl::maximum_number_of_pts); ++n2) {
```

Comment:
I feel this entire test would be more legible if written as, e.g.
```
// define at beginning of test
const size_t lgl_max = Basis::lgl::maximum_number_of_pts;

// rewrite every loop to use this local variable
for (size_t n0 = min_extents; n0 <= std::min(6_st, lgl_max); ++n0) {
}
```
but I also don't feel strongly one way or the other.